apps/i18n/common/en_us.json

```diff
@@ -2114,6 +2126,7 @@
   "showWords": "Show words",
   "showingLibrariesFromSection": "Showing libraries from section: ",
   "signup": "Sign up for the intro course",
+  "signOutButton": "Sign out",
```

Comment:
Is there an existing string for this? (I see it in English and Spanish so there's something that gets translated that means Sign Out)

Reply:
I don't think there is a "Sign Out" in this file already.

Reply:
No, there is no established string for "Sign out" available to the front-end.